web/src/copy_and_paste.ts

```diff
@@ -365,23 +389,26 @@ export function paste_handler_converter(paste_html) {
         },
     });
     turndownService.addRule("strikethrough", {
-        filter: ["del", "s", "strike"],
+        // eslint-disable-next-line @typescript-eslint/consistent-type-assertions
+        filter: ["del", "s", "strike"] as TagName[],
```

Comment:
Here the filter from TurndownService uses the `keyof HTMLElementTagNameMap` (HTMLElementTagNameMap comes from TypeScript's `lib.dom.d.ts`) in which "strike" and "math" does not exist.
Since I cannot change the type of Filter, I would like some guidance on what to do for this.